### PR TITLE
Add bio for chauhpriya

### LINKED ISSUE #22: Add a bio for chaupriya
- [x] Create branch
- [x] Create a chaupriya.md file
- [x] Open a pull request
- [ ] get and receive feedback
- [ ] Merge the pull request

### PR DESCRIPTION
**Pull request by chauhpriya**
Resolves #22 that issue requests a bio for chauhpriya, this pull request provides

### CODE REVIEW

File: chauhpriya.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Chauhpriya
+
```
Comment (severkemc) [inline chauhpriya.md:2]: @chauhpriya You didn't include enough detail: what was the last movie you watched.

Comment (nakanm2) [inline chauhpriya.md:2]: @chauhpriya What is your favorite color? :)

**Comment by chauhpriya** [comment]
@severkemc how does it look now, added more detail

**Comment by chauhpriya** [comment]
@nakanm2 : Added color

**Pull request merged**